trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/induced_drag_aircraft.py

```diff
@@ -42,18 +50,95 @@ def induced_drag_aircraft(state,settings,geometry):
     wing_e        = geometry.wings['main_wing'].span_efficiency
     ar            = geometry.wings['main_wing'].aspect_ratio # TODO: get estimate from weissinger
     CDp           = state.conditions.aerodynamics.drag_breakdown.parasite.total
+    wingleth      = 0.   #winglet length #geometry.wings['main_wing'].wingleth

```

Comment:
change comment, this shouldn't be the name for storage in the high level data structure